src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -6,43 +6,36 @@ namespace D2L.Bmx.Okta;
 
 internal interface IOktaSessionStorage {
 	void SaveSessions( List<OktaSessionCache> sessions );
-	List<OktaSessionCache> Sessions();
+	List<OktaSessionCache> GetSessions();
```

Comment:
naming nit - to make it clearer that this is a read operation.
I've seen some libraries that have a method/function named `thing()` that actually creates a `thing`.